utils/compression.sql

```diff
@@ -0,0 +1,42 @@
+-- Update metadata for chunks compressed before 2.0.
+--
+-- After you have updated from pre-2.0 it might be that some
+-- compressed chunks are missing information about the approximate row
+-- count. To correct that, call this function. Note that it will only
+-- update approximate row counts that are NULL, so running it several
+-- times will not overwrite existing values.
+--
+-- To avoid blocking too many tables, it will commit after each chunk,
+-- which means that the function might be slow if there is a lot of
+-- data to update, but it will not block other queries.
+CREATE PROCEDURE correct_compression_chunk_size ()
```

Comment:
I think I'd call this `fix_compression_row_count_stats`, given that that size is a separate thing with calculate here.

Reply:
Name changed.